reference/7.4/PSReadLine/Set-PSReadLineOption.md

```diff
@@ -25,7 +25,8 @@ Set-PSReadLineOption [-EditMode <EditMode>] [-ContinuationPrompt <String>] [-His
  [-HistorySearchCaseSensitive] [-HistorySaveStyle <HistorySaveStyle>] [-HistorySavePath <String>]
  [-AnsiEscapeTimeout <Int32>] [-PromptText <String[]>] [-ViModeIndicator <ViModeStyle>]
  [-ViModeChangeHandler <ScriptBlock>] [-PredictionSource <PredictionSource>]
- [-PredictionViewStyle <PredictionViewStyle>] [-Colors <Hashtable>] [<CommonParameters>]
+ [-PredictionViewStyle <PredictionViewStyle>] [-Colors <Hashtable>] [-TerminateOrphanedConsoleApps]
+ [-ProgressAction <ActionPreference>] [<CommonParameters>]
```

Comment:
`-ProgressAction` is part of `CommonParameters`, isn't it?

Reply:
Oops, right. PlatyPS added that because it doesn't know about it.